samples/net/sockets/http_get/README.rst

```diff
@@ -45,6 +45,31 @@ and dumps the response. You can edit the source code to issue a request
 to any other site on the Internet (or on the local network, in which
 case no NAT/routing setup is needed).
 
+Enabling TLS support
+=================================
+
+In order to enable TLS support in the sample, you need to build the project with
+``overlay-tls.conf`` overlay config file enabled, for example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/
+   :board: qemu_x86
+   :conf: "prj.conf overlay-tls.conf"
+   :goals: build
+   :compact:
+
+An alternative way is to specify ``-DOVERLAY_CONFIG=overlay-tls.conf`` when
+running cmake.
+
+Certificate used by the sample can be found in the ``src`` directory. The
+certificate was selected to enable access to the default website configured in
+the sample (https://google.com). In order to access different webpage over TLS
```

Comment:
Change to:

    To access a different web page over TLS, provide an appropriate certificate to
    authenticate to that web server.